src/parseUsdl.js

```diff
@@ -13,13 +13,21 @@ exports.parse = function parseCode128(str, options = defaultOptions) {
     if (!started) {
       if (line.indexOf('ANSI ') === 0) {
         started = true
+
+        // has DLDAQ
+        if(line.indexOf('DLDAQ') !== -1) {

```

Comment:
slightly simpler, self-explanatory (non-blocking)
```suggestion
        if(line.includes('DLDAQ')) {
```

Reply:
love it - I'll push a change for this